eth_crowdfund_be/eth_crowdfund_api/migrations/versions/23e7fe5df48f_.py

```diff
@@ -0,0 +1,59 @@
+"""empty message
+
+Revision ID: 23e7fe5df48f
+Revises: 
+Create Date: 2020-09-03 15:51:58.781746
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '23e7fe5df48f'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
to make modifications to the database schema,  first run `python manage.py db migrate` then run `python manage.py db upgrade`. Similar to migrations that have an `up` and a `down` function